optuna_dashboard/ts/components/GraphHistory.tsx

```diff
@@ -69,6 +69,20 @@ export const GraphHistory: FC<{
       theme.palette.mode,
       markerSize
     )
+    const element = document.getElementById(plotDomId)
+    if (element != null && studies.length === 1) {
+      // @ts-ignore
+      element.on("plotly_click", function (data) {
+        const link =
+          URL_PREFIX +
+          `/studies/${studies[0].id}}/trials?numbers=${data.points[0].x}`
```

Comment:
[nits] It seems to have redundant `}` here.

Reply:
As a result, the url became something like `http://localhost:8080/dashboard/studies/0%7D/trials?numbers=0`, where `%7D` comes from `}`.